crates/ruff_python_formatter/src/other/except_handler_except_handler.rs

```diff
@@ -49,32 +51,42 @@ impl FormatNodeRule<ExceptHandlerExceptHandler> for FormatExceptHandlerExceptHan
         write!(
             f,
             [
-                text("except"),
-                match self.except_handler_kind {
-                    ExceptHandlerKind::Regular => None,
-                    ExceptHandlerKind::Starred => Some(text("*")),
-                }
-            ]
-        )?;
+                clause_header(
+                    ClauseHeader::ExceptHandler(item),
+                    dangling_comments,
+                    &format_with(|f| {
```

Comment:
The formatting of the case header has been moved into the `format_with`. The `clause_header` implementation now takes care of writing the `:` and the trailing colon comments.